### DISCUSSION TITLE
Is LaTeX rendered correctly in this situation?

### DISCUSSION THREAD

**Discussion by FluffiLyn** [community/community#143280] [Repositories] [answered]
When the code block is near a character, it won’t be correctly rendered. For example:

- 欧姆定律$R=\frac{U}{I}$

However, if I add an blank between them, it can be rendered correctly.

- 欧姆定律 $R=\frac{U}{I}$

The former one can be rendered in vscode using “Markdown All in One” plugin, but cannot on the github webpage. I wonder whether this is a bug or just designed this way.

**Comment by github-actions** [comment]
🕒 **Discussion Activity Reminder** 🕒

This Discussion has been labeled as dormant by an automated system for having no activity in the last 60 days. Please consider one the following actions:

1️⃣ Close as Out of Date: If the topic is no longer relevant, close the Discussion as `out of date` at the bottom of the page.

2️⃣ Provide More Information: Share additional details or context — or let the community know if you've found a solution on your own.

3️⃣ Mark a Reply as Answer: If your question has been answered by a reply, mark the most helpful reply as the solution.

Note: This dormant notification will only apply to Discussions with the `Question` label. To learn more, see our [recent announcement](https://github.com/orgs/community/discussions/70478).

Thank you for helping bring this Discussion to a resolution! 💬

**Comment by nishanmudalige** [comment, marked as answer]
Try using MathJax, this might solve your issue:

```
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
```

See more at: https://www.mathjax.org/#gettingstarted
